CONTRIBUTING.md

```
@@ -41,8 +41,8 @@ In CI, these are passed via GitHub Secrets.
 
 ## Running Tests
 
-> **Note:** Test credentials must be set as environment variables before 
-> running tests. See [Test Credentials](#test-credentials) above.
+> **Note:** Test credentials must be set before running tests. 
+> See [Configuration](#configuration) above.
```

markdownlint-cli2 (0.23.0)

[warning] Link fragments should be valid

(MD051, link-fragments)